src/modules/users/Config.cpp

```diff
@@ -23,6 +23,17 @@
 #include <QFile>
 #include <QRegExp>
 
+#ifdef HAVE_ICU
+#include <unicode/translit.h>
+#include <unicode/unistr.h>
+
+//Did my best choosing compound ID
```

Comment:
The comment should probably do two things
 - explain briefly what this does (define a sequence of transliterations that are applied)
 - explain that  this is an initial attempt at doing the transliteration